python/lsst/sims/catUtils/mixins/VariabilityMixin.py

```diff
@@ -2,13 +2,24 @@
 import linecache
 import math
 import os
+import copy
 import json as json
 from lsst.sims.catalogs.measures.instance import register_class, register_method, compound
 from scipy.interpolate import InterpolatedUnivariateSpline
 from scipy.interpolate import UnivariateSpline
```

Comment:
copy is used to cache the random number generator (there are calls to copy.deepcopy() in the AGN caching code)